### PR TITLE
Added various missing Danish translations

### PR DESCRIPTION
**Pull request by kaspernj**

### CODE REVIEW

File: config/locales/da.yml [hunk 1]
```diff
@@ -109,20 +109,20 @@ da:
...
         name: Navn
       spree/state_change:
-        state_changes:
-        state_from:
-        state_to:
-        timestamp:
-        type:
-        updated:
-        user:
+        state_changes: Stautsændringer
```
Comment (kaspernj) [inline config/locales/da.yml]: Typo

File: config/locales/da.yml [hunk 2]
```diff
@@ -157,47 +157,47 @@ da:
...
             inventory_unit:
               other_completed_return_item_exists:
             reimbursement:
-              cannot_be_associated_unless_accepted:
+              cannot_be_associated_unless_accepted: Skal være accepteret med det associerede objekt.
         spree/store:
           attributes:
             base:
-              cannot_destroy_default_store:
+              cannot_destroy_default_store: Kan ikke slette standardbutikken
```
Comment (kaspernj) [inline config/locales/da.yml]: Missing space

Comment (kaspernj) [inline config/locales/da.yml]: Fixed in 93e8694

File: config/locales/da.yml (hunk 1 above)
Comment (kaspernj) [inline config/locales/da.yml]: Fixed in 93e8694

**Comment by tvdeyen** [comment]
Thanks, could you please squash all into one commit? It would be great if we could avoid unnecessary PR related commits in the history.

**Comment by priyank-gupta** [comment]
:+1: 
@kaspernj Are you still available to squash these commits ?

**Pull request merged**